css/style.css

```diff
@@ -16,4 +15,101 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+header {
+  height: 80px;
```

Comment:
The font size in header may change, would it be better if we use relative unit (em/rem) to fix height?

Reply:
it will update next week (HTML/CSS week3)